huaweicloud/services/acceptance/workspace/resource_huaweicloud_workspace_desktop_test.go

```diff
@@ -154,6 +155,91 @@ func TestAccDesktop_UpdateWithEpsId(t *testing.T) {
+		resourceName = "huaweicloud_workspace_desktop.test"
+		rName        = acceptance.RandomAccResourceNameWithDash()
+	)
+
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&desktop,
+		getDesktopFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccdesktop_powerAction(rName, "os-stop", "SOFT"),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(resourceName, "name", rName),
+					resource.TestCheckResourceAttr(resourceName, "power_action", "os-stop"),
+					resource.TestCheckResourceAttr(resourceName, "power_action_type", "SOFT"),
+					resource.TestCheckResourceAttr(resourceName, "status", "SHUTOFF"),
+				),
+			},
+			{
+				Config: testAccdesktop_powerAction(rName, "os-start", "SOFT"),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(resourceName, "name", rName),
+					resource.TestCheckResourceAttr(resourceName, "power_action", "os-start"),
+					resource.TestCheckResourceAttr(resourceName, "power_action_type", "SOFT"),
+					resource.TestCheckResourceAttr(resourceName, "status", "ACTIVE"),
+				),
+			},
+			{
+				Config: testAccdesktop_powerAction(rName, "reboot", "SOFT"),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(resourceName, "name", rName),
+					resource.TestCheckResourceAttr(resourceName, "power_action", "reboot"),
+					resource.TestCheckResourceAttr(resourceName, "power_action_type", "SOFT"),
+					resource.TestCheckResourceAttr(resourceName, "status", "ACTIVE"),
+				),
+			},
+			{
+				Config: testAccdesktop_powerAction(rName, "reboot", "HARD"),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(resourceName, "name", rName),
+					resource.TestCheckResourceAttr(resourceName, "power_action", "reboot"),
+					resource.TestCheckResourceAttr(resourceName, "power_action_type", "HARD"),
+					resource.TestCheckResourceAttr(resourceName, "status", "ACTIVE"),
+				),
+			},
+			{
+				Config: testAccdesktop_powerAction(rName, "os-hibernate", "HARD"),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(resourceName, "name", rName),
+					resource.TestCheckResourceAttr(resourceName, "power_action", "os-hibernate"),
+					resource.TestCheckResourceAttr(resourceName, "power_action_type", "HARD"),
+					resource.TestCheckResourceAttr(resourceName, "status", "SHUTOFF"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
+					"delete_user",
+					"image_type",
```

Comment:
why are `delete_user`, `image_type` and `vpc_id` missed in Import section of docs?

Reply:
fixed